### PR TITLE
Lex/sidebar

### PR DESCRIPTION
**Pull request by LPeterson18**
1. Added App.tsx file
2. Added sidebar component

### CODE REVIEW

File: src/client/App.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  width: '100%',
+  height: '100%',
+  display: 'table' as 'table',
+  tableLayout: 'fixed' as 'fixed'
+}
+
+export const column = {
+  minHeight: '100%',
+  display: 'table-cell'
+}
```
Comment (thehobbit85) [inline src/client/App.tsx]: This doesn't seem to be used anywhere.

File: src/client/components/Sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React, { PureComponent } from 'react'
+
+import Sidetab from './Sidetab'
+
+export const divider = {
+  marginTop: '15px',
+  width: '72%',
+  borderTop: '.5px solid white'
+}
```
Comment (thehobbit85) [inline src/client/components/Sidebar.tsx]: Is that used anywhere?

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
See comments

**Comment by LPeterson18** [comment]
Removed unnecessary styling

**Review by thehobbit85 — APPROVED** [review]

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**